utils/ruleset-whitelist.csv

```diff
@@ -1,3527 +1,3562 @@
-b326680f30a9e718cfab124855675c6b1d30fd7dc77ce1bcee1994aeb7747451,0,1,0,Crakpass.xml
-a558785d006e95550a34d309e23f47f79ab251a2b90085a06ef290f9c2a06120,1,1,0,Crash-Space.xml
-37b61cf19660f1b656869745c08b76899015d6028a19629b792555b06115bed9,0,1,0,Creative_Little_Readers.com-falsemixed.xml
-6ee9eab4d6c93aeb2754c6b34e841dc18fd6a0a2c8d59f7d546e6db23bb56c52,1,1,0,CreativePartnershipsAustralia.xml
-04f0d43bd1ab894b64d8ca6fb5e81b0213a4905c1bbd19bf92700ccada4fc0e4,1,1,0,Criminal_Justice_Inspectorates.xml
-b79d2673063115a953fdbc430f120a2798a09c573c1adfb592b23f0bf05bc411,1,1,0,Crittercism.com.xml
-1cd80467b55158721959b770308681c9e59a37977c7a4581996bb359e14bcd8d,1,1,0,Croscill.com.xml
-6d62cfeac329c56785a2de8ed1378f036aa820828ef36c0a94ddbc2f591cb1d5,0,1,0,Cross-Pixel-Media-problematic.xml
-8b9817171a2f4284abfa8220e171d15c012a0a7e0fbf0167ccf4cf7e305e66ee,1,1,0,Cross-Pixel-Media.xml
-221dcf657c72d6e40f1a51a91d5ecd09044d123c12802befb0c64e122824d1a3,1,1,0,Crowd-Science.xml
-c027ff29586919b8f684e70f870491c7465d14a49c4d7f4b3d9a891959d006eb,1,0,0,CruisersForum.com.xml
-80ff66109e0038eecb6e67362f210c6d389b69ab1044cf621d1ea28f7b624f19,1,1,0,Cryout_Creations.eu.xml
-5c03af15958cb4c8708e01cba17343bc6ff5960e3e4ea5f8b079e6198ecd4080,0,1,0,Cryptalloy.de-problematic.xml
-734c847c64dd4746922d7dfd8874f2de5aeb8db469d6165352965bfc43a7d564,1,0,0,Cryptomonkeys.com.xml
-cb23a1bd35b86d242a4028ed7f0d6fa55a1b01d620b22f552c54ab3e8aa64562,0,1,0,CrySyS-Lab.xml
-372abfbdb2705be196421bfc03d1c8b87e4bd0fda032ac674fbbaed328278fa8,1,1,0,CSC.xml
-c5c09f646294a95e637a5f083cf0a011f0bdca6a4603d1d102973457588acd74,0,1,0,CSDb.dk.xml
-27a6af8836f65d1739e68b58c32ab2feb9d4567c111f0ecaf5c95d20e6442040,1,1,0,CSIS.org.xml
-cbd89ebf0dda04e2e724d28aa789e60106497b41b01593359e833372e862e31d,1,0,0,CTEP-EBP.com.xml
-7ad4d3c27fac03a257936dd9c380abec312479338110714cb22eb95085035d35,0,1,0,CTOvision.com.xml
-c176d8858c1e88f8d843269a3f3f99ad7350b4633a7af01212c959df51ac0673,1,1,0,Ctrl_Alt_Del.xml
-4c82fb71dd76ecce008a283a688f468dc95758a8b164fd4daac98b747eb1a7a8,0,1,0,Ctt.xml
-417001159887bb430ea3c8e00dc43c7e3b6b1b895e889674a4869c467d6ca0f7,0,1,0,CUFP.org.xml
-1458bcaba2c8a2abacdb39524deb5de8016181e03ae8030d8933478fb054ee1b,0,1,0,Cultura.no.xml
-8619973ac3fe5c8a8e4ddd2d2ba0c23acc05568d7ddb6921b33a04ee6cd05cdc,1,1,0,Cumulus_Networks.xml
-e60cd38f5563cab988912843568ef8da87077845d86b102a1c738ed39d4b3e61,0,1,0,Cupid_plc.com.xml
-075ebb3dfb5aa302007e743e7d34d4a2ada1db88aa6b7875a2e14361eae60ed8,1,1,0,Cupid.xml
-a1375ad5b72a20bf94ccf118cc4d2386590687196810774dbd35f29de8b058fd,1,0,0,Curbed.com.xml
-5631d37146e3333b9b113ff5ff7dd6cadfac322d6dd31d5f82a625a911363658,1,1,0,Curso_de_Italiano.xml
-94d9f71dfa4e323b8aa8420abd459c4e11a852fc300dae5da00b91af9c61eb36,1,1,0,CustomWheelConnection.com.xml
-ecc5224bdf13e70d53652f6f917ddaf3525b293a3b014583a7e366f04851145c,1,0,0,cwrap.org.xml
-ecb6609b54104e170eccfb8b213b46ef8571ed159691b69cd4e38ef28887120b,1,1,0,Cyando.xml
-62f9443ccedff68a2edfe57cbd28c734eaaa8a90
```

Comment:
Please remove this line.

Reply:
That's the last line in [ruleset-whitelist.csv](https://github.com/EFForg/https-everywhere/pull/17012/files#diff-eb28a567e434b28fd29f644e942de4a6).  Looks like a line added by `git`.  Not sure why the `github` preview in the `conversation` tab for this PR is showing that I've commented on another line.